### DISCUSSION TITLE
Default collapsed state for Table Panel

### DISCUSSION THREAD

**Discussion by jamest-nz** [filamentphp/filament#5273] [Help]
Hi there, in the current version of Tables I have a collapsible panel, working well like this:

`Panel::make([ Stack::make([ $stackedFieldsInAPanel,  ]), ])->collapsible()`

Is there a way to specify the Panel's default collapsed/uncollapsed state for each row in the table? The basic idea is that the user could have certain records set as "favorites" and those rows would be expanded by default to show a little more info.

Perhaps something like a parameter `->collapsed( ... )` on Panel, which accepts a closure that returns the initial collapsed state on a row-by-row basis.

Is there any way to set the initial collapsed state of a Panel at the moment?

Thanks

**Comment by danharrin** [comment]
No way at the moment but I'd accept a PR :)
